tests/run-make/thumb-none-qemu/rmake.rs

```diff
@@ -0,0 +1,51 @@
+//! How to run this
+//! $ ./x.py clean
+//! $ ./x.py test --target thumbv6m-none-eabi,thumbv7m-none-eabi tests/run-make
+//!
+//! For supported targets, see `example/.cargo/config`
```

Comment:
that file notes that some targets are not quite accurate because qemu is too old. Maybe that has changed? we can try that separately though.

Reply:
Unfortunately, I have no clue what the up-to-date cargo config in this case is for thumb targets and I don't have the environment to set it, I'm fine with keeping the config as-is for now. If anyone knowledgeable about thumb can update that config, I would be happy to accept.

Reply:
I've now updated the `thumbv6m-none-eabi` target to use the `-cpu cortex-m0`. We'll see if that causes issues, it shouldn't if the qemu version is remotely recent. Otherwise I think that config file looks fine.